cities/laupheim.json

```diff
@@ -0,0 +1,199 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                    9.883797,
+                    48.229994
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Laupheimer Wochenmarkt",
+                "location": "Unterer Marktplatz Laupheim",
+                "opening_hours": "Th,Sa 07:00-13:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Fastnachtsmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": "2017 Feb 23 08:00-18:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
```

Comment:
We do not yet support special rendering for markets at the same position, see #68. Therefore, some of the markets you added won't be shown even when "Alle" is selected. They are simply hidden behind another marker which has a higher z-index.
A quick solution to the problem would be if you could just shift the coordinates a tiny bit if there is already another market with the same coordinates.